search/custom-embeddings/custom_embeddings.ipynb

```diff
@@ -0,0 +1,800 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "z3Gi-zdCeEbE"
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2024 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "WcfbGhs1eJF6"
+   },
+   "source": [
+    "# Document Q&A With Retrieval Augmented Generation\n",
```

Comment:
Suggest adding to the title (or a subtitle) something about "Vertex AI Search" and "custom embeddings" to differentiate the notebook from the other RAG notebooks that we have.

Reply:
Missed that, this notebook was originally copied from another notebook!